app/models/game.rb

```diff
@@ -1,4 +1,63 @@
 class Game < ActiveRecord::Base
   belongs_to :users
```

Comment:
The problem is that I think belongs_to is a one-to-one association. My guess is that we will need to eventually set up separate associations for belongs_to player 1 and belongs_to player 2, and link the players to the users somehow - but we may just not have gotten to that yet. This would be a great thing to pick Ilya's brain about.